CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ if (CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG" OR CMAKE_BUILD_TYPE_UC STREQUAL "RELWIT
 endif()
 
 option(ENABLE_TESTS "Provide unit_test_dbms target with Google.Test unit tests" ON)
-option(ENABLE_EXAMPLES "Build all example programs in 'examples' subdirectories" OFF)
+option(ENABLE_EXAMPLES "Build all example programs in 'examples' subdirectories" ON)
```

Comment:
Debugging leftover?

Reply:
It was left on purpose to test the examples on the CI. Now that we're building all examples for AMD_DEBUG, this is not needed anymore. Also, I've added the examples of KMS and Storage to ensure they're built and linked correctly.